applications_tests/gls_navier_stokes_3d/poiseuille3d-flow-control_gls_bdf1.output

```diff
@@ -6,99 +6,89 @@ Running on 1 MPI rank(s)...
 *******************************************************************************
 Transient iteration: 1        Time: 0.1      Time step: 0.1      CFL: 0       
 *******************************************************************************
-+------------------------------------------+
-|  Flow control summary                    |
-+------------------------------------------+
-Inlet area : 3.06147
-Flow rate : -10.0508
-Beta applied : 70
+---------------------
+Flow control summary
+---------------------
+Space-average velocity: -0
```

Comment:
It was very unclear if it's "space-average" or "space-averaged". I looked on Google and It seems space-average is often used, but that's true that space-averaged sounds better.

Also, I use space-averaged because there's also the average velocities concept which is the time-averaged velocity and I wanted to distinguish the two more explicitly.